index.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+    color: gray;
+}
```

Comment:
- [ ] It is a good practice to use class selector as a reference rather than element selector. This is for semantics reason. Kindly use the class name from your index file to style your `h1` content.